src/components/tasks/card/index.tsx

```diff
@@ -85,7 +85,7 @@ const Card: FC<Props> = ({
   
   function handleProgressColor(percentCompleted: number, startedOn: string, endsOn: string): string {
     const percentageofDays = getPercentageOfDays(startedOn, endsOn)
-    if(percentageofDays >= percentCompleted) {
+    if(percentCompleted === 100 && (percentageofDays >= percentCompleted || percentageofDays < 0)) {
```

Comment:
Hey can you add this percentageofDays < 0 condition to all the other if statements which are defining the color

Reply:
In my opinion, percentageofDays<50,percentageofDays<25 already handles it. Am I missing something?
![image](https://user-images.githubusercontent.com/75324135/185832225-e4bba44f-086e-4c76-84b1-07f96fa8e055.png)

Reply:
@YashJain24-chief Please paste the link to the code, not a screenshot of the isolated code.

Reply:
https://github.com/Real-Dev-Squad/website-status/blob/1b14c606288aaeef7e40f60af852b7684ba81f03/src/components/tasks/card/index.tsx#L88